content/events/2023-civo-navigate.md

```diff
@@ -0,0 +1,12 @@
+title = "Civo Navigate Europe 2023"
+date = "2023-08-21T00:00:01Z"
+description = "Exciting times await you at the Civo Navigate Europe 2023 event this September! Find us at our vibrant booth. We also have a line-up of engaging sessions waiting for you: a workshop, a breakout session, and your chance to explore the dynamic world of WebAssembly with us. Bring your passion, your questions, and your energy. And remember, amidst all the learning, there’s also room for some fun!"
+
+[extra]
+type = "event"
+eventdates = "September 5th & 6th - London, UK"
```

Comment:
Huge nitpick but for the AI one we used the `26th-27th` format rather than `26th & 27th`

Reply:
I think we should use "-" bc some events will span more than 2 days.

Reply:
Thanks @karenhchu and I will also abbreviate the month, as you have done (so the new events all match).